packages/react-components/lib/doors/door-panel.tsx

```diff
@@ -68,11 +68,11 @@ const useStyles = makeStyles((theme) => ({
   },
   buttons: {
     backgroundColor: theme.palette.primary.main,
-    color: theme.palette.getContrastText(theme.palette.primary.main),
+    color: theme.palette.primary.contrastText,
```

Comment:
These are not properly applied to the button texts, the class is applied to `ButtonGroup` which does not seem to propgate to the buttons.

Reply:
Hm okay will look at this and above comment on themes